libexec/cli/instance.list.exec

```diff
@@ -92,12 +104,24 @@ for i in $DAEMONS; do
             fi
         fi
 
-        printf "%-16s %-8s %s\n" "${FILE_NAME}" "${DAEMON_PID}" "${DAEMON_IMAGE:-}"
+        if [[ "${JSONFMT}" == "true" ]]; then
+          if [ "$COUNT" -lt "$DLEN" ]; then
+            printf '\t{\n\t\t"instance":"%s",\n\t\t"pid":"%s",\n\t\t"img":"%s"\n\t},\n' "${FILE_NAME}" "${DAEMON_PID}" "${DAEMON_IMAGE:-}"
```

Comment:
Would it be useful to save this big gnarly string in a variable so that you don't have to repeat it on 2 lines?